include/libtorrent/session_handle.hpp

```diff
@@ -811,19 +811,33 @@ namespace libtorrent {
 		// the swarm. This operation cannot fail. When it completes, you will
 		// receive a torrent_removed_alert.
 		//
+		// remove_torrent() is non-blocking, but will remove the torrent from the
+		// session synchronously. Calling session_handle::add_torrent() immediately
+		// afterward with the same torrent will succeed.
```

Comment:
maybe it would make sense to mention here that adding the torrent back immediately will still return a different `torrent_handle` (i.e. not resurrect the previous one, even if it was still valid)